run-ci.sh

```diff
@@ -33,4 +33,6 @@ if [ ${RUN_TEST=1} -eq 1 ]; then
         echo "[INFO] Running test $ex"
         (cd tests && dub --temp-build --single $ex $DUB_FLAGS)
     done
+    # check the keylog file was written
+    test -f "tests/loggedkeys.txt"
```

Comment:
I would also move this to the test ?

Reply:
Don't we care that the file is written after the program exits?

Reply:
Moved to the main function in the test.